ui/src/app/pages/data/cohort-review/cohort-review-list-item.tsx

```diff
@@ -82,37 +87,72 @@ export const CohortReviewListItem = fp.flow(
       <React.Fragment>
         <div
           style={{
-            border: `1px solid ${colorWithWhiteness(colors.black, 0.8)}`,
-            borderRadius: '2px',
+            border: `1px solid ${colorWithWhiteness(colors.black, 0.9)}`,
+            borderRadius: '3px',
+            boxShadow:
+              '0 0 2px 0 rgba(0,0,0,0.12), 0 3px 2px 0 rgba(0,0,0,0.12)',
             display: 'flex',
-            height: '2.25rem',
+            height: '3.5rem',
             marginTop: '0.25rem',
           }}
         >
           <div
             style={{
-              borderRight: `1px solid ${colorWithWhiteness(colors.black, 0.8)}`,
               flex: '0 0 10%',
-              padding: '10px 0 10px 10px',
             }}
           >
-            <ResourceActionsMenu actions={actions()} disabled={readOnly} />
+            <RadioButton
+              style={{ marginLeft: '0.25rem' }}
+              name='reviewItem'
+              onChange={() => onSelect(cohortReview.cohortReviewId)}
+              checked={selected}
+            />
           </div>
           <div
             style={{
               color: colors.primary,
-              flex: '0 0 90%',
-              fontSize: '13px',
+              flex: '0 0 50%',
+              lineHeight: '0.75rem',
               padding: '0.5rem 0.25rem',
             }}
           >
-            <RadioButton
-              style={{ marginRight: '0.25rem' }}
-              name='reviewItem'
-              onChange={() => onSelect(cohortReview.cohortReviewId)}
-              checked={selected}
-            />
-            {cohortReview.cohortName}
+            <div style={{ fontSize: '14px', fontWeight: 600 }}>
+              {cohortReview.cohortName}
+            </div>
+            <Clickable>
+              <span
+                style={{
+                  color: colors.accent,
+                  fontSize: '12px',
+                  fontWeight: 500,
+                }}
+              >
+                Cohort details
+              </span>
+            </Clickable>
```

Comment:
Will add navigation to this link with [RW-8742](https://precisionmedicineinitiative.atlassian.net/browse/RW-8742).